_pages/en/saas-and-cloud-strategy.md

```diff
@@ -0,0 +1,230 @@
+---
+layout: default
+title: SaaS vs Cloud Strategy
+lang: en
+ref: documents
+status: posted
+permalink: /saas-vs-cloud.html
+---
+
```

Comment:
In Context, also refer to:

1) NIST definition: https://www.nist.gov/publications/nist-definition-cloud-computing

This is where all of the IaaS, PaaS, SaaS you here in all literature came from

2) Buy vs Build

We need to DRILL in people's head that the equation is not that simple. 

It's ALWAYS ALWAYS Buy AND Build.

Better yet, how you framed it as Obtain and Build.